src/engine/strat_engine/blockdev.rs

```diff
@@ -7,7 +7,7 @@
 use std::fs::OpenOptions;
 use std::path::PathBuf;
 
-use time::Timespec;
+use chrono::{DateTime, Local};
```

Comment:
Yes. I'm worried that that might have been an oversight. It's nice that the chrono crate makes that decision more explicit.